src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const types of actions) {
+    switch (types.type) {
+      case 'addProperties':
+        for (const addKey in types.extraData) {
+          state[`${addKey}`] = types.extraData[`${addKey}`];
+        }
```

Comment:
there's a special method for that - Object.assign(), or you can use `...` spread operator. Both would be better than a loop